database/mssql/scripts/dbscripts/PSP_PIMS_S30_00/PIMS S30.00 Change Log.txt

```diff
@@ -1,3 +1,8 @@
+PSP_PIMS S30.01 Sprint 30 2022-Jun-10
```

Comment:
The changes are part of s30.00 vs s30.01

Reply:
Yes, I was able make the changes in 30.00.  30.01 has noit bneen created.